publish/publish.proj

```diff
@@ -53,6 +53,26 @@
   <Target Name="PublishDebFilesToDebianRepo"
           DependsOnTargets="GenerateDebRepoFiles"
           Condition="'@(DebInstallerFile)' != ''">
+      <ItemGroup>
+        <RepoIds Include="DebianId_ubuntu1404-x64">
+          <Key>$(DebianId_ubuntu1404-x64)</Key>
+        </RepoIds>
+        <RepoIds Include="DebianId_debian8-x64">
+          <Key>$(DebianId_debian8-x64)</Key>
+        </RepoIds>
+        <RepoIds Include="DebianId_ubuntu1604-x64">
+          <Key>$(DebianId_ubuntu1604-x64)</Key>
+        </RepoIds>
+        <RepoIds Include="DebianId_ubuntu1610-x64">
+          <Key>$(DebianId_ubuntu1610-x64)</Key>
+        </RepoIds>
+      </ItemGroup>
+      <PropertyGroup>
+        <RidWithoutDots>$([System.String]::Copy($(DistroRid)).Replace('.', ''))</RidWithoutDots>
```

Comment:
Since the keys themselves come from PropertyGroups, it looks like those can't have dots